bases/shared/entrypoint.sh

```diff
@@ -739,6 +744,31 @@ case "$ROLE" in
         sed -i.bak '/^\[state-sync]/,/^\[/{s/^snapshot-interval[[:space:]]*=.*/snapshot-interval = 0/}' "$AGORIC_HOME/config/app.toml"
         start_chain --pruning everything
         ;;
+    "follower")
+        if [[ ! -f "$AGORIC_HOME/data/agoric/swingstore.sqlite" ]]; then
+            cd /state/
+            if [[ ! -f "/state/agoric_11796108.tar.lz4" ]]; then
+                wget -O agoric_11796108.tar.lz4 https://storage.googleapis.com/agoric-snapshots-public/agoric_11796108-polkachu/agoric_11796108.tar.lz4 --inet4-only
+            fi
+            apt update
+            apt install lz4
+            lz4 -c -d agoric_11796108.tar.lz4  | tar -x -C $AGORIC_HOME
+            wget -O addrbook.json https://storage.googleapis.com/agoric-snapshots-public/agoric_11796108-polkachu/addrbook.json
+            cp -f addrbook.json "$AGORIC_HOME/config/addrbook.json"
```

Comment:
I'm actually not sure if this specific one is available anymore, and as of last I checked the latest is [`11810177`](https://polkachu.com/tendermint_snapshots/agoric) (maybe old by the time you read this too). Note that Polkachu deletes old snapshots and only keeps most recent one.

We should either:

1. Make this configurable so that we can specify what snapshot to download. A non-upgrade redeployment would involve bumping this to the latest.
2. Have a script determine the last available snapshot and use that.

I think 1 is easiest and likely to have less issues during upgrades (e.g. latest still would require upgrade).

Reply:
I am getting a new full-node export from mathieu to use here. I will upload it in a gcloud storage and make it configurable after more testing.

Reply:
I have made the downloadable link configurable and pointing towards polkachu snapshots because they are uploading recent snapshots reliably for long time now.